### PR TITLE
Cleaning out Incident word from frontend

### PR DESCRIPTION
**Pull request by Ukochka**
**What this PR does**:

**Which issue(s) this PR fixes**:

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [x] `CHANGELOG.md` updated

### CODE REVIEW

File: grafana-plugin/src/containers/IncidentMatcher/IncidentMatcher.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ const IncidentMatcher = observer((props: IncidentMatcherProps) => {
       <div className={cx('columns')}>
         <div className={cx('incident-list')}>
           <Text.Title className={cx('title')} level={5}>
-            Matching Incidents
+            Matching Alert groups
```
Comment (joeyorlando) [inline grafana-plugin/src/containers/IncidentMatcher/IncidentMatcher.tsx:66]: should the "g" in "groups" also be capitalized if we're going to capitalize "Alert"?

File: grafana-plugin/src/plugin.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
   "includes": [
     {
       "type": "page",
-      "name": "Incidents",
+      "name": "Alert groups",
```
Comment (joeyorlando) [inline grafana-plugin/src/plugin.json:34]: same w/ capitalization here.

File: grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx [hunk 3]
```diff
@@ -50,8 +50,8 @@ class SettingsPage extends React.Component<SettingsPageProps, SettingsPageState>
         <div className={cx('settings')}>
           <Field
             loading={!teamStore.currentTeam}
-            label="Require resolution note when resolve incident"
-            description={`Once user clicks "Resolve" for an incident they are require to fill a resolution note about the incident`}
+            label="Require resolution note when resolve alert group"
+            description={`Once user clicks "Resolve" for an alert group they are require to fill a resolution note about the alert group`}
```
Comment (joeyorlando) [inline grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx:54]:
```suggestion
            description={`Once user clicks "Resolve" for an alert group, they will be required to fill in a resolution note about the alert group`}
```

File: grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx [hunk 4]
```diff
@@ -50,8 +50,8 @@ class SettingsPage extends React.Component<SettingsPageProps, SettingsPageState>
         <div className={cx('settings')}>
           <Field
             loading={!teamStore.currentTeam}
-            label="Require resolution note when resolve incident"
-            description={`Once user clicks "Resolve" for an incident they are require to fill a resolution note about the incident`}
+            label="Require resolution note when resolve alert group"
```
Comment (joeyorlando) [inline grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx:53]:
```suggestion
            label="Require a resolution note when resolving alert groups"
```

**Review by joeyorlando — COMMENTED** [review]
I think we should consider talking to the docs team (or maybe we have a copy-writing team?) regarding capitalization consistency (ex. `Alert group` vs `alert group` vs `Alert Group`). We have instances of all three throughout the UI.

File: grafana-plugin/src/pages/incident/Incident.tsx [hunk 5]
```diff
@@ -395,7 +395,7 @@ class IncidentPage extends React.Component<IncidentPageProps, IncidentPageState>
             </li>
           ))}
         </ul>
-        <Field label="Leave a resolution note" description="Will also show up in the thread of incident in Slack">
+        <Field label="Leave a resolution note" description="Will also show up in the thread of alert group in Slack">
```
Comment (teodosii) [inline grafana-plugin/src/pages/incident/Incident.tsx:398]: Shouldn't this _be show up in the thread of **the** alert group_ or _should up in the thread of **an** alert group_ ? It is not correct as it is right now

File: grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx (hunk 4 above)
Comment (teodosii) [reply grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx:53]: ^ good mention

File: grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx (hunk 3 above)
Comment (teodosii) [reply grafana-plugin/src/pages/settings/tabs/MainSettings/MainSettings.tsx:54]: ^ same here

**Review by teodosii — APPROVED** [review]
Approved but please add those changes.

**Comment by teodosii** [comment]
This should also have the change in the changelog I believe, it's good to keep track of all the frontend changes this way.

**Pull request merged**